client/src/test/java/TestRunner.bat

```diff
@@ -0,0 +1,11 @@
+@echo off
+
+SET ClientDesktop=../../../build/libs/com.fiskaly.kassensichv.client.desktop-0.0.1-alpha.jar
```

Comment:
Shouldn't we stick to our "naming convention", i.e. `clientGeneral`...?

Reply:
The .jar file is called *.client.desktop.* . What should be changed now? The jar name or the variable name?

Reply:
Oh... the jar file should also be named `general` instead of `desktop` to be consistent.

Reply:
Changed to general 5d5ad9f5a2a30bca402d0a43e8063d8796a6491a